docs/source/notebooks/api_quickstart.ipynb

```diff
@@ -16,12 +16,13 @@
      "name": "stdout",
```

Comment:
<ul><li>Can get rid of %matplotlib inline</li><li>Let's replace all seaborn calls by ArviZ ones?</li><li>Maybe organize imports alphabetically, as in other NBs we modified recently?</li></ul><pre class="ql-syntax" spellcheck="false">import arviz as az
import matplotlib.pyplot as plt
import numpy as np
import pymc3 as pm
import theano.tensor as tt
</pre><ul><li>Let's use ArviZ style ;) And in another cell below (this has to do with how MPL sets defaults):</li></ul><pre class="ql-syntax" spellcheck="false">%config InlineBackend.figure_format = 'retina'
az.style.use('arviz-darkgrid')
print('Running on PyMC3 v{}'.format(pm.__version__))
</pre><ul><li>You can look at the NBs in the <a href="https://github.com/pymc-devs/resources" target="_blank">resources repo</a> for an example of how we format NBs.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>